regolith/builders/recentcollabsbuilder.py

```diff
@@ -47,7 +47,16 @@ def construct_global_ctx(self):
     def latex(self):
         rc = self.rc
         since_date = dt.date.today() - relativedelta(months=48)
-        person = fuzzy_retrieval(all_docs_from_collection(rc.client, "people"), ['aka', 'name', '_id'], self.rc.people, case_sensitive = False)
+        if self.rc.people:
+            target = self.rc.people
+        else:
+            raise ValueError(
+                "No default person."
+                "Use the '--people' in CLI to specify the person."
+            )
+        person = fuzzy_retrieval(all_docs_from_collection(rc.client, "people"), ['aka', 'name', '_id'], target, case_sensitive=False)
+        if not person:
+            raise ValueError("There is no doc associated with person '{}'".format(target))
```

Comment:
I think this needs to be clearer.  What does "There is no doc associated with person" mean to a user?